snuba/query/processors/helpers.py

```diff
@@ -0,0 +1,37 @@
+from snuba.query.conditions import (
```

Comment:
please give this file a name that provides some context. Like performance_expressions.py or something similar. If we call it helpers it will become the catch all file where to throw anything and we won't be able anymore to define a responsibility for this file.